src/main/java/uk/ac/sanger/sccp/stan/model/reagentplate/ReagentPlateLayout.java

```diff
@@ -5,15 +5,15 @@
 import java.util.Objects;
 
 /**
- * A specification for a type of {@link ReagentPlate}
+ * A specification for a layout of {@link ReagentPlate}
  * @author dr6
  */
-public class ReagentPlateType {
+public class ReagentPlateLayout {
     private final String name;
     private final int numRows, numColumns;
 
-    public ReagentPlateType(String name, int numRows, int numColumns) {
-        this.name = "Dual index plate";
+    public ReagentPlateLayout(String name, int numRows, int numColumns) {
+        this.name = name;
```

Comment:
Before name  was getting initialized to"Dual index plate" in constructor and it is changed now. Hope it will be set properly from where the instance is created

Reply:
Yes, the way it was before looks like a mistake.